app/events/service.py

```diff
@@ -151,4 +153,4 @@ def _save_to_events_collection(event: Event):
 
     :param event: Event to save
     """
-    firestore_db.collection(EVENTS_COLLECTION).add(event.to_dict())
+    firestore_db().collection(EVENTS_COLLECTION).add(event.to_dict())
```

Comment:
Why is this a function again? I know it's because of the tests but I'm still not clear on why

Reply:
If we just have it as a variable, it runs not when you _use_ the variable, but when you _import_ the file. So even if you just need the collection name, not even the client, it will still run.

The problem is, it need to first have an initialized firebase app running. This is a headache to deal with when testing, I need to import stuff but have to manage where the import line is vs. where I actually started the app.

Since the firebase module already caches the client (you can test by calling the function twice and checking the object references with `is`.) it will always be the same anyways.